### ISSUE TITLE
Quest Broken: "Mutiny on the Mercy"

### ISSUE THREAD

**Issue by Th3w8er** [dalaranwow/dalaran-wow#5398]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

Quest Broken: "Mutiny on the Mercy", on the lower floor of the boat there should be mobs to kill which grant the 5 Barrels of Blasting Powder. There are no mobs, instead they are spawned in a big group outside the ship, on an ice shelf just north of the small island where You board the ship.

**Expected behaviour**: Tell us what should happen instead

The mobs should be on the lower floor of the boat, seperated between the cannons.

**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Golganeth** [comment]
Duplicate of #457
